misc/experimental/rust/celix_bindings/build.rs

```diff
@@ -44,19 +58,15 @@ fn main() {
     println!("cargo:info=Start build.rs for celix_bindings");
     let include_paths = print_include_paths().unwrap();
```

Comment:
Missing `parse_callbacks(Box::new(bindgen::CargoCallbacks))` so that changed Celix C headers will not lead to rebuild of this crate.